examples/src/bin/tlsserver-mio.rs

```diff
@@ -429,6 +429,8 @@ Options:
                         to certificate.  Optional.
     --auth CERTFILE     Enable client authentication, and accept certificates
                         signed by those roots provided in CERTFILE.
+    --crl CRLFILE ...   Perform client certificate revocation checking using the DER-encoded
+                        CRLFILE. May be used multiple times.
```

Comment:
It probably makes sense to error if `--crl` is provided and not `--auth` - I haven't done that in this branch to keep things simple while we figure out some other more interesting bits.

Reply:
I added this quickly since it was a small change ultimately.